src/ol/format/gpxformat.js

```diff
@@ -569,7 +573,17 @@ ol.format.GPX.writeWptType_ = function(node, coordinate, objectStack) {
     default:
       // pass
   }
-  var orderedKeys = ol.format.GPX.WPT_TYPE_SEQUENCE_[namespaceURI];
+  var orderedKeys;
+  switch (node.nodeName) {
+    case 'rtept':
+      orderedKeys = ol.format.GPX.RTEPT_TYPE_SEQUENCE_[namespaceURI];
+      break;
+    case 'trkpt':
+      orderedKeys = ol.format.GPX.TRKPT_TYPE_SEQUENCE_[namespaceURI];
```

Comment:
since the keys are the same for RTEPT and TRKPT, is there really a need for these to be separate? Or can they be combined into 1 sequence instead?